transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTcpMd5Test.java

```diff
@@ -54,7 +54,7 @@ public void setup() {
         server = (EpollServerSocketChannel) bootstrap.group(GROUP)
                 .channel(EpollServerSocketChannel.class)
                 .handler(new ChannelInboundHandlerAdapter())
-                .bind(new InetSocketAddress(0)).syncUninterruptibly().channel();
+                .bind(new InetSocketAddress(NetUtil.LOCALHOST4, 0)).syncUninterruptibly().channel();
```

Comment:
What I'm suggesting would `s/LOCALHOST4/LOCALHOST/g` in this file provide any benefit?